requirements.txt

```diff
@@ -1,7 +1,7 @@
 # pypi requirements
 accelerate>=0.12
 datasets>=1.11.0
-diffusers>=0.6.0
+diffusers<=0.6.0
```

Comment:
is this upper bound really needed?

Reply:
Temporarily yes, `diffusers` main builds are failing as I wrote above. See here: https://github.com/huggingface/diffusers

Reply:
Also here I fixed the thing that we actually do support diffusers < 0.6.0